config/java-regexp.header.02

```diff
@@ -0,0 +1,18 @@
+^/{95}$
```

Comment:
if config/java-regexp.header.01 and config/java-regexp.header.02 are same, lets 
just reuse `${checkstyle.regexp.header.file}` and put it in config twice.